sys-apps/flashrom/flashrom-1.1.ebuild

```diff
@@ -0,0 +1,161 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://review.coreboot.org/flashrom.git"
+	inherit git-r3
+else
+	MY_P="${PN}-v${PV}"
+	SRC_URI="https://download.flashrom.org/releases/${MY_P}.tar.bz2"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86"
+	S="${WORKDIR}/${MY_P}"
+fi
+
+DESCRIPTION="Utility for reading, writing, erasing and verifying flash ROM chips"
+HOMEPAGE="https://flashrom.org/"
+
+LICENSE="GPL-2"
+SLOT="0"
+# The defaults match the upstream Makefile.
+# Note: Do not list bitbang_spi as it is not a programmer; it's a backend used
+# by some other spi programmers.
+IUSE_PROGRAMMERS="
+	atahpt
+	+atapromise
+	+atavia
+	+buspirate_spi
```

Comment:
There's a bug open about naming of these USE flags, 
https://bugs.gentoo.org/694664

you think you could deal with this bug in this PR? Seems like a simple bash substitution is enough to make them work in the ebuild.

Reply:
I'll do it =)

Reply:
It was a little bit trickier but looks to be done.

Reply:
Ooops..

I've not worked on `<flashrom-1.1`. Should we update older ebuilds?

Reply:
Also it seems I have to update `metadata.xml`. I will do it tomorrow.

Reply:
Yeah, I didn't realize flashrom had this many ebuilds in the tree. Maybe you can start by cleaning them up, then update what's left. Sigh, it requires some work to do for sure. Here's what I'd do: 
bump this without fixing those issues, then clean older versions leaving 1.1, 1.0 and 0.9.8 in the tree, THEN edit them all and metadata.xml in one commit in a new PR.

Reply:
Finally I drop changes related to underscores.

I'll deal with https://bugs.gentoo.org/694664 in another request after this one.